logos/logos-workernode/logos_worker_node/logos_bridge.py

```
@@ -1196,10 +1196,16 @@ async def _execute_stream_command(self, ws, cmd_id: str, params: dict[str, Any])
             )
         finally:
+            # Decrement before aclose() so that a client-side disconnect that
+            # leaves httpx draining the upstream stream does not keep
+            # worker_active > 0 and falsely trigger proxy_stuck detection.
+            await lane_manager.decrement_active_requests(lane_id)
             if upstream is not None:
                 try:
-                    await upstream.aclose()
+                    await asyncio.wait_for(upstream.aclose(), timeout=5.0)
                 except Exception:  # noqa: BLE001
                     pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)